### PR TITLE
Add script for seal integration

### PR DESCRIPTION
**Pull request by binaryberry**
This script _should_ trigger the seal when seal me up is mentioned in a room that has the seal setup (cf. the [seal route](https://github.com/binaryberry/seal/blob/master/server.rb#L15)). 

Not sure whether it works nor how to test this so any ideas, let me know :-)

cc @tijmenb

### CODE REVIEW

**Comment by tijmenb** [comment]
This seems plausible :) I will try to find a way we can actually test this.

Perhaps it would be good to add a shared secret between this app and seal, so that we won't have to worry about other people triggering a bark. This app could post to `https://moody-seal.herokuapp.com/teams/#{room}?secret=#{process.env.SEAL_SECRET}`, and seal would only act on requests where `params[:secret] == ENV["SEAL_SECRET"]`.

**Comment by tijmenb** [comment]
You can test this manually by running `bin/hubot`. This will install the dependencies and give you a command line hubot which acts like hubot on Slack. You can then test the integration by saying `hubot seal me up`.

File: scripts/seal.coffee [hunk 1]
```diff
@@ -0,0 +1,9 @@
+module.exports = (robot) ->
+  robot.hear /seal me up/i, (res) ->
```
Comment (Davidslv) [inline scripts/seal.coffee:2]: I would add `^` to the regular expression `/^seal me up/i`so it doesn't catch it in a middle of a phrase.

Although I'm not sure where someone would use that in a middle of a sentence... 

`Last night I seal me up` :D

Comment (binaryberry) [inline scripts/seal.coffee:2]: Good point! I need to get back to this work... I have to make adjustments to the seal too. On my list!

**Comment by Davidslv** [comment]
ping ping :)

**Comment by binaryberry** [comment]
Hello both! I've finally gone back to this and followed your advice, and I'm much more confident this should work since when I try it in `bin/hubot` the seal is triggered as expected. 

I've also prepared changes to the seal in [this PR](https://github.com/binaryberry/seal/pull/111), adding the shared secret. The branch with these changes is currently deployed.

I will be emailing you the secret key so you can add it to the env vars of gds-hubot.

File: scripts/seal.coffee (hunk 1 above)
Comment (binaryberry) [reply scripts/seal.coffee:2]: Hi! On second thought, I think the cases where someone might want to trigger the seal while adding a word or character to the incantation, ie. “seal me up!” or “seal me up, please” are more likely than them saying “seal me up” in a conversation without wanting to summon the seal, so I’d rather leave it like this for now and monitor how it goes, to see if making it more restrictive is necessary.

**Comment by binaryberry** [comment]
Changed to a `POST` and added `/bark` to the url following Tijmen's comments on the [Seal's PR](https://github.com/binaryberry/seal/pull/111). Should be ready now!

**Pull request merged**